scripts/ci/_utils.sh

```diff
@@ -239,8 +239,6 @@ function update_all_md5_files() {
 # files. There are the following, problems with the current Dockerfiles that need longer build times:
 # 1) We need to fix group permissions of files in Docker because different linux build services have
```

Comment:
This numbered list does not make sense now. The previous paragraph talks about many problems, and only one problem is described.

Reply:
True!